resources/styles/components/task-step/all-steps.less

```diff
@@ -9,7 +9,7 @@
   .spacer-step {
     padding: @tutor-card-body-padding;
   }
-  .interactive-step, .reading-step {
+  .interactive-step > .interactive-content, .reading-step {
```

Comment:
Can you place each selector on separate lines? Otherwise, it's easy to miss the comma.